modules/cert-manager-acme-dns01-explicit-gcp.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-explicit-gcp_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using explicit credentials for GCP Cloud DNS
+
+You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on GCP. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Google CloudDNS.
+
+.Prerequisites
+
+* You have access to a role that has permissions to create issuers, certificates, and secrets in the desired namespace.
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have set up an IAM role for Google CloudDNS. For more information, see link:https://cert-manager.io/docs/configuration/acme/dns01/google/[Google CloudDNS] in the upstream cert-manager documentation.
```

Comment:
> You have set up an IAM role for Google CloudDNS

Incorrect. For GCP, it is not only IAM role, it is Google Cloud service account of right roles.

Reply:
Doesn't GCP still use "IAM" keyword in it's service when referring to service accounts? I'd prefer "service account" though.

Reply:
> For GCP, it is not **only** IAM role

I mean it is not **ONLY** IAM role. So I suggest "Google Cloud service account of right roles"